packages/block-library/src/latest-posts/edit.js

```diff
@@ -104,16 +104,17 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 				categories && categories.length > 0
 					? categories.map( ( cat ) => cat.id )
 					: [];
-			const latestPostsQuery = pickBy(
-				{
-					categories: catIds,
-					author: selectedAuthor,
-					order,
-					orderby: orderBy,
-					per_page: postsToShow,
-					_embed: 'wp:featuredmedia',
-				},
```

Comment:
We declare the object inline, so we're pretty certain that it will not be nullish. Therefore, no special handling is needed.